auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -0,0 +1,296 @@
+    if (this.$refs.numberOfBusinessesRef.validate()) {
+      this.updateDissolutionBatchSize(this.numberOfBusinesses)
+      this.triggerEditOnOff()
+    }
+  }
+
+  /** Edit or Cancel button is clicked. */
+  triggerEditOnOff (): void {
+    this.isEdit = !this.isEdit
+    // closing the menu
+    this.menu = false
+  }
+
+  /** The array of validations rule(s) for the Dissolution Batch Size text field. */
+  get dissolutionBatchSizeRules (): Array<(v) => boolean | string> {
+    return [
+      v => !!v || 'The number of businesses to be moved into D1 dissolution per batch. Maximum of 2000.',
+      v => (v % 1 === 0) || 'The number of businesses to be moved into D1 dissolution per batch. Maximum of 2000.',
+      v => v >= 0 || 'The number of businesses to be moved into D1 dissolution per batch. Maximum of 2000.',
+      v => v <= 2000 || 'Exceeds the maximum of 2000 businesses per batch.'
+    ]
+  }
+
+  /** The action button text depending on whether the dissolution job is paused or running. */
+  get actionBtnText (): string {
+    return this.isOnHold ? 'Resume' : 'Pause'
+  }
+
+  /**
+   * If non-edit, show the number of businesses into D1 from the store.
+   * Otherwise, it'll be reactive to whatever is being typed in the text field.
+   */
+  get scheduleSummaryNumber (): number {
+    return this.isEdit ? this.numberOfBusinesses : this.getDissolutionBatchSize()
+  }
+
+  /** Emit whether on hold or not to the parent. */
+  @Emit('update:onHold')
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  onHoldUpdate (onHold: boolean): void {}
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+.section-container {
+  color: $gray9;
+
+  label {
+    color: $gray9;
+    font-weight: bold;
+  }
+}
+
+// Vuetify Override of list item
+.theme--light.v-list-item .v-list-item__action-text, .theme--light.v-list-item .v-list-item__subtitle {
+  color: $app-blue;
+  .v-icon.v-icon {
+    color: $app-blue;
+  }
+}
+
+// Adding shadows to buttons
+.action-btn, .more-actions-btn {
+    box-shadow: 0 1px 1px 0px rgb(0 0 0 / 20%), 0 2px 2px 0 rgb(0 0 0 / 14%), 0 1px 5px 0 rgb(0 0 0 / 12%);
+    -webkit-box-shadow: 0 1px 1px 0px rgb(0 0 0 / 20%), 0 2px 2px 0 rgb(0 0 0 / 14%), 0 1px 5px 0 rgb(0 0 0 / 12%);
+  }
+
+// The action buttons styling (Pause, Resume)
+.action-btn {
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  min-width: 5.5rem !important;
+  margin-right: 1px;
+}
+
+// The more actions button styling (chevron)
+.more-actions-btn {
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+// make menu button slightly smaller
+.menu-btn {
+  min-width: unset !important;
+}
+
+// Increasing dropdown menu (list) size to same size as button
+.v-list {
+  min-width: 8.5rem !important;
+}
+
+// Making the pencil icon smaller
+.mdi-pencil:before, .mdi-pencil-set {
+  font-size: 16px;
+}
+
+// Increasing the Edit text size by a bit
+.edit-txt {
+  font-size: 14px;
```

Comment:
Somewhere, we declared a base class `font-size-14` to do this in the template instead of needed CSS here. YMMV

Reply:
Awesome catch. I updated everything this across all the new files. Thanks!